indico/htdocs/js/indico/widgets/person_link_widget.js

```diff
@@ -48,12 +49,76 @@
         var $noOtherPlaceholder = $fieldDisplay.find('#no-other-placeholder-' + options.fieldId);
         var $buttonAddExisting = $('#add-existing-' + options.fieldId);
         var $buttonAddNew = $('#add-new-' + options.fieldId);
+        var $buttonAlphaOrder = $fieldDisplay.find('.alpha-order-switch');
+
+        function updateOrderAttributes($list) {
+            $list.children().each(function(i, elem) {
+                $(elem).data('person').displayOrder = i;
+            });
+        }
+
+        function setupList(entryType, $list, sortedPeople) {
+            var isSortable = options.sort == 'all' || _.find(options.sort, entryType);
+
+            $list.html('');
+
+            if (isSortable) {
+                var initialIndex;
+
+                $list.sortable({
+                    axis: 'y',
+                    containment: 'parent',
+                    cursor: 'move',
+                    distance: 10,
+                    handle: '.sort-handle',
+                    items: '> .person-row',
+                    tolerance: 'pointer',
+                    forceHelperSize: true,
+                    forcePlaceholderSize: true,
+                    helper: function(e, item) {
+                        var originals = item.children();
+                        var helper = item.clone();
+                        helper.children().each(function(i) {
+                            $(this).width(originals.eq(i).width());
+                        });
+                        return helper;
+                    },
+                    start: function(e, ui) {
+                        initialIndex = ui.item.index();
+                    },
+                    update: function(e, ui) {
+                        var newIndex = ui.item.index(),
+                            person = $(ui.item).data('person');
```

Comment:
I think we have a mixture of both patterns. I'm trying to find out a good analysis or at least a standard.